frontend/src/Components/ToastDelete/ToastDelte.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Swal from 'sweetalert2';
+
+const ToastDelete = ({ onConfirm }) => {
```

Comment:
This component not good, should separate between toast and button, or this is a component button, should change name